backend/app/api/api_v1/endpoints/health.py

```
@@ -0,0 +1,104 @@
+    overall_healthy = True
+
+    # Check database connectivity and version
+    try:
+        result = await db.execute(text("SELECT VERSION()"))
+        db_version = result.scalar()
+        components["database"] = {
+            "status": "healthy",
+            "version": db_version,
+            "type": "MySQL"
+        }
+    except Exception as e:
```

Ruff (0.14.2)

Do not catch blind exception: `Exception`

(BLE001)